### PR TITLE
move vcf merging to after bootstrapann

### PR DESCRIPTION
**Pull request by Lucpen**
## PR checklist

- [X] move vcf merging to after bootstrapann
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/genomic-medicine-sweden/tomte/tree/master/.github/CONTRIBUTING.md)
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Check for unexpected warnings in debug mode (`nextflow run . -profile debug,test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: subworkflows/local/allele_specific_calling.nf [hunk 1]
```diff
@@ -34,25 +40,75 @@ workflow ALLELE_SPECIFIC_CALLING {
...
+            ch_ind_vcf_tbi,
             GATK4_ASEREADCOUNTER.out.csv
         )
         ch_versions = ch_versions.mix(BOOTSTRAPANN.out.versions.first())
 
+        TABIX_BGZIP(BOOTSTRAPANN.out.vcf)
+        ch_versions = ch_versions.mix(TABIX_BGZIP.out.versions.first())
+
+        BCFTOOLS_INDEX_2(TABIX_BGZIP.out.output)
+        ch_versions = ch_versions.mix(BCFTOOLS_INDEX_2.out.versions.first())
```
Comment (jemten) [inline subworkflows/local/allele_specific_calling.nf:64]: If you want to you can use the `TABIX_BGZIPTABIX` module to produce [ meta, vcf.gz, vcf.gz.tbi ] directly. That would however require an update to how the files are collected downstream. We can wait with this as well

File: subworkflows/local/allele_specific_calling.nf [hunk 2]
```diff
@@ -34,25 +40,75 @@ workflow ALLELE_SPECIFIC_CALLING {
...
+        RENAME_FILES( ch_case_vcf.single)
+
+        BCFTOOLS_MERGE.out.merged_variants
+            .mix( RENAME_FILES.out.output )
+            .set { ch_vcf }
+
+        TABIX_TABIX( ch_vcf )
+        ch_tbi = TABIX_TABIX.out.tbi
+
+        ch_versions = ch_versions.mix( BCFTOOLS_MERGE.out.versions.first() )
```
Comment (jemten) [inline subworkflows/local/allele_specific_calling.nf:106]: Let's add all versions to the end.

**Review by jemten — APPROVED** [review]
Very minor comments. Feel free to ignore if you want to start the test as soon as possible

File: subworkflows/local/allele_specific_calling.nf (hunk 2 above)
Comment (Lucpen) [reply subworkflows/local/allele_specific_calling.nf:106]: Done

**Pull request merged**